src/styled/Supplier/AppSupplierData/StyledSpan.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+const StyledSpan = styled.span`
```

Comment:
Same situation as above, consider using native Chakra props instead.
Please get familar with Chakra [theming](https://chakra-ui.com/docs/theming/theme), in this case Chakra is providing [typography solution](https://chakra-ui.com/docs/theming/theme#typography) for you.